pandas/core/common.py

```diff
@@ -479,6 +481,45 @@ def _dict_compat(d):
                 for key, value in iteritems(d))
 
 
+def prep_maping_for_to_dict(into):
+    """
+    Helper function to standardize the supplied mapping so it can
+    be passed to the ``Series.to_dict()`` and ``DataFrame.to_dict()``
+    
+    .. versionadded:: 0.21.0
+
+    Parameters
+    ----------
+    into : instance or subclass of collections.Mapping
+        The argument supplied to ``to_dict``.  Must be a class, an
```

Comment:
Agreed, though you can add a `See Also` section to point to `to_dict`. You can also remove references to `to_dict` in the summary.